web-client/views/info.jade

```diff
@@ -0,0 +1,221 @@
+                          dd.syntheticHaploinsufficiency.col-sm-9.sgdSource
+                          dt.col-sm-3 Synthetic Lethality:
+                          dd.syntheticLethality.col-sm-9.sgdSource
+                          dt.col-sm-3 Synthetic Rescue:
+                          dd.syntheticRescue.col-sm-9.sgdSource
+                      dl.row
+                        h5.col-sm-3 Total Interactions:
+                        dd.totalInteractions.sgdSource
+                .card
+                  #geneOntologyHeading.card-header
+                    h5.mb-0
+                      a.green(data-toggle='collapse', data-parent='#accordion', href='#geneOntology', aria-expanded='true', aria-controls='collapseOne')
+                        | Gene Ontology
+                  #geneOntology.collapse
+                    .card-block
+                      dl.row
+                        dt.col-sm-3 Gene Summary:
+                        dd.geneSummary.col-sm-9.sgdSource
+                        dt.col-sm-3 Molecular Function
+                        dd.molecularFunction.col-sm-9.sgdSource
+                        dt.col-sm-3 Biological Process:
+                        dd.biologicalProcess.col-sm-9.sgdSource
+                        dt.col-sm-3 Cellular Component:
+                        dd.cellularComponent.col-sm-9.sgdSource
+                .card
+                  #sourcesHeading.card-header
+                    h5.mb-0
+                      a.violet(data-toggle='collapse', data-parent='#accordion', href='#sources', aria-expanded='true', aria-controls='collapseOne')
+                        | Sources
+                  #sources.collapse
+                    .card-block
+                      dl.row
+                        a#sgdSource.col-6.col-md-4.sgd-link.database-link
+                      dl.row
+                        a#uniprotSource.col-6.col-md-4.uniprot-link.database-link
+                      dl.row
+                        a#ensemblSource.col-6.col-md-4.ensembl-link.database-link
+                      dl.row
+                        a#ncbiSource.col-6.col-md-4.ncbi-link.database-link
+                      dl.row
+                        a#jasparSource.col-6.col-md-4.jaspar-link.database-link
+        iframe#iframeFooter(src='http://dondi.github.io/GRNsight/onlyfooter.html', scrolling='no')
+        script(src='https://code.jquery.com/jquery-3.2.1.min.js', integrity='sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=', crossorigin='anonymous')
+        script(src='https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js', integrity='sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4', crossorigin='anonymous')
+        script(src='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js', integrity='sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1', crossorigin='anonymous')
+        script(src='gene/api.js')
+        script(src='gene/info.js')
```

Comment:
Nicely done. More of a "transcription" rather than a "rewrite," I think. Still, a good chunk of work.